### PR TITLE
add usage readme, allow usage under Windows

### PR DESCRIPTION
**Pull request by igorko**

### CODE REVIEW

File: bestEnclosingRect/MaxRectsBinPack.cpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /** @file MaxRectsBinPack.cpp
-	@author Jukka Jyl�nki
+	@author Jukka Jyl�nki
```
Comment (stefanbeller) [inline bestEnclosingRect/MaxRectsBinPack.cpp:2]: The name of the original author is broken here

**Comment by igorko** [comment]
I will add a fix tomorrow

**Comment by stefanbeller** [comment]
I squashed your commits into one, ommitting the first one (which was originally authored by Jukka Jylänki jujjyl@gmail.com), and cherry picked the result.

**Comment by igorko** [comment]
cool. Thanks

**Pull request closed without merging**